openradar/utils.py

```diff
@@ -312,6 +312,8 @@ def consistent_product_expected(prodcode, timeframe):
     can be used to determine if a product needs to be published, or
     not. If not, the rescaled equivalent should be published.
     """
+    if prodcode == 'u' and timeframe == 'fh':
```

Comment:
Ik denk dat je `timeframe in 'fh'` bedoelt in plaats van `timeframe == 'fh'`, anders gaan de ultimate producten nooit gerescaled worden. Verder denk ik dat deze twee if-statements wel in één keer kunnen:
```python
 if prodcode in 'au' and timeframe in 'fh':
     return True
```
Als jij dat ook denkt, dan graag aanpassen.